src/lib/flexbox/api/flex-offset.ts

```diff
@@ -84,7 +103,43 @@ export class FlexOffsetDirective extends BaseFxDirective implements OnInit, OnCh
   // Protected methods
   // *********************************************
 
+  /** The flex-direction of this element's host container. Defaults to 'row'. */
+  protected _layout = 'row';
```

Comment:
Now I understand. We do have value validators. I think you found a potential bug since we are not using that here NOR checking for row-reverse or column-reverse.

I will fix.